### ISSUE TITLE
NTR: Peptide binding term for other peptide hormones in the glucose regulation pathway

### ISSUE THREAD

**Issue by cerivs** [geneontology/go-ontology#13143]
Peptide hormone binding has a child insulin binding.  However insulin is only part of the equation there are other peptide hormones that regulate glucose and cell's response to insulin.  
Some grouping terms for vertebrates similar to the insect specific adipokinetik hormone binding would be helpful in annotating gene function.  
Glucagon cholecytokinin, somatostatin, pancreatic polypeptide and other peptide hormones are all vital in the body's response to  sugar.  Malfunction of any of these will cause disease in humans so being able to indicate a protein binds to these hormones is important.  I am not sure what to call this term or if we should have individual peptide hormone binding terms.  
I'm not sure what grouping terms would be helpful for other verts so not making a specific term request.

Minimal term request
glucagon binding
 Interacting selectively and non-covalently with glucagon, a polypeptide hormone involved in glucose response.  It is produced by pancreatic alpha cells and raises the concentration of glucose in the blood. 

somatostatin binding 
 Interacting selectively and non-covalently with somatostatin, a polypeptide hormone involved in regulating pancreatic alpha and pancreatic beta cells and  controlling growth hormone  secretion  as well as many  other functions.  Somatostatin is produced by several cell types including pancreatic delta cells.  There are several different mature forms of somatostatin. PMID 20472043

**Comment by cerivs** [comment]
Oops another request.
Glucagon-like peptide binding
Interacting selectively and non-covalently with glucagon-like peptide, a antihyperglycemic hormone.  Glucagon-like peptide is derived from the glucagon gene produced by pancreatic  alpha cells.

**Comment by krchristie** [comment]
@ukemi - Just wanted to check if it is OK within current GO guidelines to add  more binding terms for these specific peptide hormones.

**Comment by hdrabkin** [comment]
I was thinking of bring this up as an "editors-discuss"; I could see where all of these could go in a has_input slot rather than having a separate term for each.

**Comment by krchristie** [comment]
Exactly, was thinking the same thing...

**Comment by ukemi** [comment]
+1

**Comment by krchristie** [comment]
Some annotation numbers from AmiGO for "**peptide hormone binding (GO:0017046)**":

Counts (selected) by evidence type:
- (162)	experimental evidence
- (129)	similarity evidence used in manual assertion

Counts for GO class (direct):
- (289)	peptide hormone binding
-  (52)	insulin binding
-  (21)	corticotropin-releasing hormone binding
-  (10)	calcitonin binding
-   (1)	adipokinetic hormone binding

Counts (selected) by species:
- (101)	Rattus norvegicus
-  (56)	Mus musculus
-  (43)	Homo sapiens
-  (38)	Oncorhynchus mykiss
-  (18)	Bos taurus
-  (18)	Canis lupus familiaris
-  (13)	Sus scrofa
-  (10)	Gallus gallus
-   (8) 	Danio rerio

**Comment by krchristie** [comment]
Summary of 3/16/2017 ontology editors meeting: While we think that eventually this will be better handled by putting the specific peptide hormone in an annotation extension, we recognize that currently some groups are not able to do this, so we will make these terms.

**Comment by krchristie** [comment]
Here are the terms:

Note that I guessed that GOC:cvs was the correct id for you for the definition dbxref. If I guess incorrectly, let me know and I'll correct it.

 [Term]
+id: GO:0120022
+name: glucagon binding
+namespace: molecular_function
+def: "Interacting selectively and non-covalently with glucagon, a polypeptide hormone involved in glucose response. It is produced by pancreatic alpha cells and raises the concentration of glucose in the blood." [GOC:cvs]
+intersection_of: GO:0017046 ! peptide hormone binding
+intersection_of: has_input CHEBI:5391 ! glucagon
+created_by: kchris
+creation_date: 2017-03-17T14:46:29Z
+
+[Term]
+id: GO:0120023
+name: somatostatin binding
+namespace: molecular_function
+def: "Interacting selectively and non-covalently with somatostatin, a polypeptide hormone involved in regulating pancreatic alpha and pancreatic beta cells and controlling growth hormone secretion as well as many other functions. Somatostatin is produced by several cell types including pancreatic delta cells. There are several different mature forms of somatostatin." [GOC:cvs, PMID:20472043]
+intersection_of: GO:0017046 ! peptide hormone binding
+intersection_of: has_input CHEBI:64628 ! somatostatin
+created_by: kchris
+creation_date: 2017-03-17T14:54:21Z
+
+[Term]
+id: GO:0120024
+name: glucagon-like peptide binding
+namespace: molecular_function
+def: "Interacting selectively and non-covalently with glucagon-like peptide, a antihyperglycemic hormone. Glucagon-like peptide is derived from the glucagon gene produced by pancreatic alpha cells." [GOC:cvs]
+is_a: GO:0017046 ! peptide hormone binding
+created_by: kchris
+creation_date: 2017-03-17T14:57:26Z

**Comment by krchristie** [comment]
As indicated in https://github.com/geneontology/go-ontology/pull/13187 
- I fixed the relationship issues in my terms and also removed an explicit subclass relationship for the previously existing term 'corticotropin-releasing hormone binding' since the reasoner inferred that subclass.
- I did not remove the asserted subclass for 'insulin binding' because when I tried this (multiple times), the reasoner did not assert the subclass, so I just left it the way it was. I'm not going to look into this further.
